arm-network/2017-06-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,2812 @@
+                    },
+                    {
+                        "$ref": "#/parameters/VirtualHubParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/GatewayParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ExpressRouteConnection"
+                        },
+                        "description": "Parameters supplied to create or update a ExpressRoute connection to a scalable ExpressRoute gateway."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the ExpressRoute connection created or updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ExpressRouteConnection"
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. ExpressRoute connection is not yet provisioned; follow the Location header to poll for final outcome."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            },
+            "patch": {
+                "operationId": "ExpressRouteConnections_Update",
+                "x-ms-examples": {
+                    "ExpressRouteConnectionUpdate": { "$ref": "../examples/ExpressRouteConnectionUpdate.json" }
+                },
+                "description": "Updates an ExpressRoute connection.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/VirtualHubParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/GatewayParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+              
```

Comment:
please confirm whether these will be fully qualified ARM resource Ids? Whenever an ARM resource is referenced by another resource, it must be done using the ARM resource id. since you are doing it at many places in this spec, pleae confirm.